tests/publiccloud/transfer_repos.pm

```diff
@@ -39,6 +39,13 @@ sub run {
 
         # We need to exclude embargoed incidents
         my @all_repos = split(/,/, get_var('MAINT_TEST_REPO'));
+        my @excluded_repos = split(/,/, get_var('EXCLUDED_TEST_REPO', ''));
+        for my $exclude (@excluded_repos) {
+            for my $index (reverse 0 .. $#all_repos) {
```

Comment:
`reverse` not strictly needed, no difference if not present :slightly_smiling_face:

Reply:
i don't think so , it modifies `@all_repos` in-place , so if you remove first lower index item it will change indexes of all next items ..

Reply:
At any loop you change the inner `for` data, renewed at any time: what if you try it in a local simulation snippet?